
This reverts:
----
commit ca58e594da2486c1d28e7ad547d82266604ec4ce
Author: Peng Chen <pengchen@qti.qualcomm.com>
Date:   Fri Aug 30 17:09:52 2013 +0800

    Bluetooth: Add a new PID/VID 0cf3/e005 for AR3012.
----

as it breaks AR3011 device

https://bugs.mageia.org/show_bug.cgi?id=13597

Signed-off-by: Thomas Backlund <tmb@mageia.org>

diff -Nurp linux-3.15.2/drivers/bluetooth/ath3k.c linux-3.15.2.rev/drivers/bluetooth/ath3k.c
--- linux-3.15.2/drivers/bluetooth/ath3k.c	2014-06-08 21:19:54.000000000 +0300
+++ linux-3.15.2.rev/drivers/bluetooth/ath3k.c	2014-06-27 10:01:34.611286516 +0300
@@ -90,7 +90,6 @@ static const struct usb_device_id ath3k_
 	{ USB_DEVICE(0x0b05, 0x17d0) },
 	{ USB_DEVICE(0x0CF3, 0x0036) },
 	{ USB_DEVICE(0x0CF3, 0x3004) },
-	{ USB_DEVICE(0x0CF3, 0x3005) },
 	{ USB_DEVICE(0x0CF3, 0x3008) },
 	{ USB_DEVICE(0x0CF3, 0x311D) },
 	{ USB_DEVICE(0x0CF3, 0x311E) },
@@ -140,7 +139,6 @@ static const struct usb_device_id ath3k_
 	{ USB_DEVICE(0x0b05, 0x17d0), .driver_info = BTUSB_ATH3012 },
 	{ USB_DEVICE(0x0CF3, 0x0036), .driver_info = BTUSB_ATH3012 },
 	{ USB_DEVICE(0x0cf3, 0x3004), .driver_info = BTUSB_ATH3012 },
-	{ USB_DEVICE(0x0cf3, 0x3005), .driver_info = BTUSB_ATH3012 },
 	{ USB_DEVICE(0x0cf3, 0x3008), .driver_info = BTUSB_ATH3012 },
 	{ USB_DEVICE(0x0cf3, 0x311D), .driver_info = BTUSB_ATH3012 },
 	{ USB_DEVICE(0x0cf3, 0x311E), .driver_info = BTUSB_ATH3012 },
diff -Nurp linux-3.15.2/drivers/bluetooth/btusb.c linux-3.15.2.rev/drivers/bluetooth/btusb.c
--- linux-3.15.2/drivers/bluetooth/btusb.c	2014-06-08 21:19:54.000000000 +0300
+++ linux-3.15.2.rev/drivers/bluetooth/btusb.c	2014-06-27 10:01:34.611286516 +0300
@@ -160,7 +160,6 @@ static const struct usb_device_id blackl
 	{ USB_DEVICE(0x0b05, 0x17d0), .driver_info = BTUSB_ATH3012 },
 	{ USB_DEVICE(0x0cf3, 0x0036), .driver_info = BTUSB_ATH3012 },
 	{ USB_DEVICE(0x0cf3, 0x3004), .driver_info = BTUSB_ATH3012 },
-	{ USB_DEVICE(0x0cf3, 0x3005), .driver_info = BTUSB_ATH3012 },
 	{ USB_DEVICE(0x0cf3, 0x3008), .driver_info = BTUSB_ATH3012 },
 	{ USB_DEVICE(0x0cf3, 0x311d), .driver_info = BTUSB_ATH3012 },
 	{ USB_DEVICE(0x0cf3, 0x311e), .driver_info = BTUSB_ATH3012 },
